flow360/component/simulation/meshing_param/face_params.py

```diff
@@ -26,70 +25,37 @@ class SurfaceRefinement(Flow360BaseModel):
-        self.entities is None indicates that this is a global setting.
-        """
-        if self.entities is None and self.curvature_resolution_angle is None:
-            self.curvature_resolution_angle = 12 * u.deg
-        return self
+
+class PassiveSpacing(Flow360BaseModel):
+    """
+    Passively control the mesh spacing either through other face's meshing
+    setting or doing nothing to change existing surface mesh at all.
+    """
+
+    name: Optional[str] = pd.Field(None)
+    type: Literal["projected", "unchanged"] = pd.Field()
+    refinement_type: Literal["PassiveSpacing"] = pd.Field("PassiveSpacing", frozen=True)
+    entities: EntityList[Surface] = pd.Field(alias="faces")
 
 
 class BoundaryLayer(Flow360BaseModel):
     """
     These affects volume meshing.
-    Note:
-    - We do not support per volume specification of these settings so the entities will be **obsolete** for now.
-    Should we have it at all in the release?
-
-    - `None` entities will be expanded (or just ignored and convert to global default, depending on implementation)
-    before submission. This is supposed to be applied to all the matching entities. We allow this so that we do not
-    need to have dedicated field for global settings. This is also consistent with the `FluidDynamics` class' design.
     """
 
     name: Optional[str] = pd.Field(None)
     refinement_type: Literal["BoundaryLayer"] = pd.Field("BoundaryLayer", frozen=True)
-    type: Literal["aniso", "projectAnisoSpacing", "none"] = pd.Field(default="aniso")
-    entities: Optional[EntityList[Surface]] = pd.Field(None, alias="faces")
+    entities: EntityList[Surface] = pd.Field(alias="faces")
     # pylint: disable=no-member
     first_layer_thickness: LengthType.Positive = pd.Field(
-        description="First layer thickness for volumetric anisotropic layers."
+        description="First layer thickness for volumetric anisotropic layers for given face."
     )
-    # pylint: disable=no-member
-    growth_rate: Optional[pd.PositiveFloat] = pd.Field(
-        None, description="Growth rate for volume prism layers.", ge=1
-    )  # Note:  Per face specification is actually not supported.
-    # This is a global setting in mesher similar to curvature_resolution_angle.
-
-    @pd.model_validator(mode="after")
-    def _add_global_default_growth_rate(self):
-        """
-        [CAPABILITY-LIMITATION]
-        Add **global** default for `growth_rate`.
-        Cannot add default in field definition because that may imply it can be set per surface.
-        self.entities is None indicates that this is a global setting.
-        """
-        if self.entities is None and self.growth_rate is None:
-            self.growth_rate = 1.2
-        return self
-
-
-SurfaceRefinementTypes = Union[SurfaceRefinement, BoundaryLayer]
+
+
+SurfaceRefinementTypes = Union[SurfaceRefinement, BoundaryLayer, PassiveSpacing]
```

Comment:
I think BoundaryLayer and PassiveSpacing is not `SurfaceRefinementTypes` but `VolumeRefinementTypes`. It is sourced from surface but affects volume.

Reply:
Inlined SurfaceRefinementTypes and VolumeRefinementTypes